templates/definition/charger/ocpp-goe-charger.yaml

```diff
@@ -0,0 +1,17 @@
+template: ocpp-goe-charger
+products:
+  - brand: go-e
+    description:
+      generic: Charger V3 [OCPP]
+  - brand: go-e
+    description:
+      generic: Charger Gemini [OCPP]
+  - brand: go-e
+    description:
+      generic: Charger PRO [OCPP]
```

Comment:
Bei den bestehenden go-e Templates haben wir die go-e Wallboxen in Gruppen zusammengefasst. Ich find deinen Weg (ein Eintrag pro Gerät) deutlich schöner. Macht zwar Redundanz auf der Seite, dafür ist das navigieren und finden aber besser.

Aktuell:

![Bildschirmfoto 2024-08-07 um 22 35 04](https://github.com/user-attachments/assets/4333319f-ad3d-404c-b8a9-ddcff128660f)

Bzgl. Protokoll haben wir hier an einigen Stellen (Solax, Hoymiles, Daheimladen, ...) bereits das Muster `MARKE PRODUKT (PROTOKOLL)` bzw. `MARKE PRODUKT (via PROTOKOLL)`  in Verwendung. Das würde ich hier aus sehen. Glaube, das ` via` bringt wenig Mehrwert und kann eigentlich weg.

Bei Geräten die dann mehr als ein Protokoll untersützten (hier bpsw. Charger Gemini) würde ich der existierenden Version auch nachträglich das Protokoll verpassen. Also `go-e Charger Gemini (OCPP)` und `go-e Charger Gemini (REST)`. Dann tauchen die beiden Einträge in Doku, `evcc configure` und Config UI auch direkt untereinander auf.  Die Gruppe OCPP würd ich nicht machen. Das führt sonst dazu, dass man als Nutzer in beiden Listen alphabetisch suchen muss.

Bei Geräten, die nur eine Implementierung haben (bspw. EntraTek) kann das ` (OCPP)` im Namen raus. Ich würde das nur reinnehmen, um unterschiedliche Implementierungen unterscheidbar zu machen.

Perspektivisch können wir auch überlegen, ob wir im Template neben `brand` und `description` ein dediziertes `protocol` Enum-Feld einführen. Dann könnten wir die oben beschriebene Logik (nur anzeigen, wenn erforderlich) automatisch in Docs und Config UI umsetzen. Auch eine mehrstufige Anzeige (Tabs oder Protokoll-Auswahl-Dropdown) wäre damit möglich.

Die Doku würde nach meinem Vorschlag für `go-e` heute dann erst einmal so aussehen:

![Bildschirmfoto 2024-08-07 um 23 06 30](https://github.com/user-attachments/assets/3ad0de5b-f200-4939-b77f-132c1f9ac983)

Mit dem `protocol` Feld (s.o.) könnte man die doppelten Einträge später noch vereinfachen bzw. in der UI zusammenführen.

Reply:
> Perspektivisch können wir auch überlegen, ob wir im Template neben brand und description ein dediziertes protocol Enum-Feld einführen. Dann könnten wir die oben beschriebene Logik (nur anzeigen, wenn erforderlich) automatisch in Docs und Config UI umsetzen. Auch eine mehrstufige Anzeige (Tabs oder Protokoll-Auswahl-Dropdown) wäre damit möglich.

Fände ich glaub ich sehe hilfreich für die Übersicht.

Reply:
Ich schau mal, ob ich das mit dem separaten Feld hier im Rahmen des PRs hinbekomme.